modules/admin_manual/pages/configuration/files/encryption/encryption_configuration_quick_guide.adoc

```diff
@@ -137,22 +135,20 @@ sudo -E -u www-data php occ encryption:decrypt-all -m recovery -c yes
 {occ-command-example-prefix} app:disable encryption
 ----
 
-=== Cleanup your database
+== Cleanup your database
 
-Access your ownCloud database and remove the remaining entries 
-that haven't been automatically removed with this command:
+Access your ownCloud database and remove the remaining entries that have not been automatically removed with this command:
 
 [source,sql]
 ----
 DELETE FROM oc_appconfig WHERE appid='encryption';
 ----
 
-=== Cleanup your storage
-
-Lastly you have to delete all encryption keys on storage by running this command: 
+== Cleanup your storage
 
-(Modify the path to your data directory according to your installation)
+Lastly you have to delete all encryption keys on storage by running this command, modify the path to your data directory according to your installation:
 
+[source,console]
 ----
 find /var/www/html/owncloud/data -type d -name "files_encryption" -exec rm -R {} +
 ----
```

Comment:
This is a potentially dangerous command, which may take very long time.
This command easily destroys user data. We **must** limit to depth 1 and should add an interactive `-i` to the rm command.

Reply:
Took you comment and added a improvement.
Checked locally the result of the find command.